### DISCUSSION TITLE
Should Config._check_access really use logger.info?

### DISCUSSION THREAD

**Discussion by lewisblake** [metno/pyaerocom#939] [General]
This line 
https://github.com/metno/pyaerocom/blob/89501477f70e4d47ef994e7e2d71e6b37839683d/pyaerocom/config.py#L275

pollutes the terminal with information every time `_check_access` gets called. This always pops up using the CLI and most of the time isn't needed. Should we do something about it or leave it as it is?

**Comment by lewisblake** [comment]
Example:

```console
(pya) [lewisb@ppi-r8login-b1 pyaerocom]$ pya -V
Checking access to: /lustre/storeB/project
Checking access to: /lustre/storeB/project/aerocom
Checking access to: /lustre/storeB/project/aerocom/aerocom1
🦄 pyaerocom 0.14.0
```

**Comment by avaldebe** [comment]
maybe this should `logger.debug`?

**Reply by lewisblake** [reply]
Agreed

**Comment by lewisblake** [comment]
Coming from the lustre test - more reason to remove lustre dependency.

**Reply by lewisblake** [reply]
#766

**Comment by lewisblake** [comment]
#999
